src/components/master-user-components/master-dashboard-components/master-modal/OrganizationModal.jsx

```
@@ -14,43 +15,46 @@ const OrganizationModal = ({
   handleInputChange,
   handleOrganizationFormSubmit,
   isLoading,
+  API_BASE_URL
 }) => {
   if (!isOpen) return null;
-  const API_BASE_URL = useMemo(() => `${import.meta.env.VITE_SERVER_API_URL}/api`, []);
+  
+  const apiBaseUrl = useMemo(
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)